packages/mui-material/src/Grid/Grid.js

```diff
@@ -341,7 +377,8 @@ const Grid = React.forwardRef(function Grid(inProps, ref) {
         className={clsx(classes.root, className)}
         as={component}
         ref={ref}
-        {...other}
+        breakpoints={breakpoints.keys}
```

Comment:
Oops this will end up on the dom element, adding it to the `ownerState` bag is better.